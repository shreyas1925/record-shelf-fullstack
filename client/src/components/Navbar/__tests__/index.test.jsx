import { render,screen } from "@testing-library/react"
import { Navbar } from "../.."
import { BrowserRouter } from "react-router-dom"

describe('NavBar', () => { 
    it('should render without crashing', () => {
        render(<BrowserRouter><Navbar/></BrowserRouter> )
        const appElement = screen.getByText('My Shelf',{exact:false})
        expect(appElement).toBeInTheDocument()
    })
})

describe("Navbar Snapshot", () => {
    it("should render forming a snapshot", () => {
      const { asFragment } = render(<BrowserRouter><Navbar/></BrowserRouter>);
      expect(asFragment()).toMatchSnapshot(); 
    });
  });