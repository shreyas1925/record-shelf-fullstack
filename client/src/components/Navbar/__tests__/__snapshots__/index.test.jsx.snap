// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Snapshot should render forming a snapshot 1`] = `
<DocumentFragment>
  <header>
    <a
      href="/"
    >
      <p>
        My 
        <span>
          Record
        </span>
         Shelf
      </p>
    </a>
  </header>
</DocumentFragment>
`;
